src/character.cpp

```diff
@@ -2153,7 +2153,9 @@ ret_val<bool> Character::can_wear( const item &it, bool with_equip_change ) cons
         for( const trait_id &mut : get_mutations() ) {
             const auto &branch = mut.obj();
             if( branch.conflicts_with_item( it ) ) {
-                return ret_val<bool>::make_failure( _( "Your %s mutation prevents you from wearing your %s." ),
+                return ret_val<bool>::make_failure( is_player() ?
+                                                    _( "Your %s mutation prevents you from wearing your %s." ), branch.name(), it.type_name() :
+                                                    _( "My %s mutation prevents me from wearing this %s." ), branch.name(), it.type_name(),
```

Comment:
Have you tested this? Because I think the second operand of the ternary will evaluate to just the `type_name`. That second operand consists of some invocations of the comma operator, so it evaluates to the right operand of them.
```C++
is_player ? "...", name(), type_name() : ...
// Is parsed as
is_player ? ( "...", name(), type_name() ) : ...
// Which is the same as
is_player ?( type_name() ) : ...
```
I suggest writing it like this:
```C++
make_failure( is ? "text 1" : "text 2", additional, parameters, that, will, apply, to, both, strings )
```